modules/dnn/src/layers/convolution_layer.cpp

```diff
@@ -262,29 +264,31 @@ class ConvolutionLayerImpl CV_FINAL : public BaseConvolutionLayerImpl
         {
             if (kernel_size.size() == 3)
                 return preferableTarget == DNN_TARGET_CPU;
+            if ((backendId == DNN_BACKEND_INFERENCE_ENGINE_NN_BUILDER_2019 || preferableTarget != DNN_TARGET_MYRIAD) && blobs.empty())
```

Comment:
Why we need an extra limitation for NNBuilder and Myriad target?

Reply:
Myriad supports only ngraph::op::Constant op for weights

Reply:
I see, sorry. Just misunderstood